### DISCUSSION TITLE
Pregunta TPA 2024 Grupo 8

### DISCUSSION THREAD

**Discussion by LaSudestada** [dds-jv/foro#414] [TPA - 2024] [answered]
Buenos días profe!!

Disculpe la molestia, tenemos algunas consultas del TPA y vimos que hay una categoría para el TPA 20 y 21 pero no para el 24
Se podrá abrir una o seguimos preguntando en esta categoría de "generales"?
Ah y otra de la misma índole, cuando hagamos una consulta tenemos que indicar el grupo o el ayudante en el título?

Si puedo aprovechar este mismo mensaje pregunto:
Cómo funcionan las heladeras?
 * Uno puede abrir una heladera de la nada y sacar lo que uno quiera?
 * Cómo se descuentan las viandas que alguien retira y cómo sabés cual o cuales descontar?
 * Cómo controlas que una persona no agarre tres viandas?
 * Se abren con una tarjeta magnética o con algún tipo de llave o contraseña?
 * Son como máquinas expendedoras y las personas vulnerables tienen que pedir en la aplicación la vianda que quieren?
 * O el responsable de la heladera tambien es responsable de retirar una vianda y entregarsela a quien la solicite?

Muchas gracias!!

**Comment by flbulgarelli** [comment]
> Disculpe la molestia, tenemos algunas consultas del TPA y vimos que hay una categoría para el TPA 20 y 21 pero no para el 24
Se podrá abrir una o seguimos preguntando en esta categoría de "generales"?

Listo, ahí creamos una nueva categoría. Gracias por recordarlo. 

>  Ah y otra de la misma índole, cuando hagamos una consulta tenemos que indicar el grupo o el ayudante en el título?

No, no deberían hacer eso.

**Comment by flbulgarelli** [comment, marked as answer]
:wave: ¡Hola! Paso a responder las preguntas de la entrega:

> Cómo funcionan las heladeras?
> 
>    Uno puede abrir una heladera de la nada y sacar lo que uno quiera?

En principio, sí. En próximas entregas se profundizará sobre este aspecto, sus detalles y reglas.

> Cómo se descuentan las viandas que alguien retira y cómo sabés cual o cuales descontar?
>  Cómo controlas que una persona no agarre tres viandas?
> Se abren con una tarjeta magnética o con algún tipo de llave o contraseña?

Misma respuesta que el punto anterior: se trabajará sobre este aspecto más adelante. En esta entrega no se trabajará sobre la gestión de los alimentos y su consumo una vez ingresado en la heladera, sino el proceso que va desde la registración de quien colabora hasta el ingreso del alimento (desde el punto de vista lógico; la logística necesaria para la distribución no se atacará aún).  

> Son como máquinas expendedoras y las personas vulnerables tienen que pedir en la aplicación la vianda que quieren? O el responsable de la heladera tambien es responsable de retirar una vianda y entregarsela a quien la solicite?

Nuevamente, misma respuesta. Sin embargo, te puedo adelantar que se parecerán más a lo primero y que, por ahora, la visión es que el acceso a la heladera sea personal, sin mediación de otras partes. 

Decime si quedó claro. 

¡Saludos!

**Reply by LaSudestada** [reply]
Muchas gracias profe, quedó claro, justamente me daba la impresión de que una respuesta contestaba todas, por eso las puse todas juntas. Mil gracias!! Buen finde!
